### PR TITLE
Update the aws bootstrap script with correct version

### PR DESCRIPTION
**Pull request by ericsun95**
Update the bootstrap script with correct scala version and hbase version. Since we drop the support for 2.11 and we are moving towards hbase 2, I just change it to two by default.

### CODE REVIEW

File: geomesa-hbase/geomesa-hbase-tools/bin/bootstrap-geomesa-hbase-aws.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ GMUSER=hadoop
 # todo bootstrap from the current running location and ask the user if they want to 
 # install to /opt/geomesa ? Maybe propmpt with a default of /opt/geomesa similar to
 # how the maven release plugin works?
-GMDIR="/opt/geomesa-hbase_2.11-%%project.version%%"
+GMDIR="/opt/geomesa-hbase_2.12-%%project.version%%"
```
Comment (ericsun95) [inline geomesa-hbase/geomesa-hbase-tools/bin/bootstrap-geomesa-hbase-aws.sh:22]: Or we add a scala version property similar like the `%%project.version%%` while it seems that still don't apply well for hbase1 vs hbase2 so I just leave it "hard code“ as it is.

**Comment by ericsun95** [comment]
@ronq @jrs53 @elahrvivaz first to make change to this repo, feel free to comment and review! Thanks

File: geomesa-hbase/geomesa-hbase-tools/bin/bootstrap-geomesa-hbase-aws.sh (hunk 1 above)
Comment (elahrvivaz) [reply geomesa-hbase/geomesa-hbase-tools/bin/bootstrap-geomesa-hbase-aws.sh:22]: Ah, yes I think this is coming from the maven pom, so we already have a `scala.binary.version` [flag](https://github.com/locationtech/geomesa/blob/main/pom.xml#L56).
```suggestion
GMDIR="/opt/geomesa-hbase_%%scala.binary.version%%-%%project.version%%"
```

File: geomesa-hbase/geomesa-hbase-tools/bin/bootstrap-geomesa-hbase-aws.sh [hunk 2]
```diff
@@ -59,7 +59,7 @@ ROOTDIR="${ROOTDIR%/}" # standardize to remove trailing slash
 chown -R $GMUSER:$GMUSER ${GMDIR}
 
 # Configure coprocessor auto-registration
-DISTRIBUTED_JAR_NAME=geomesa-hbase-distributed-runtime-hbase1_2.11-%%project.version%%.jar
+DISTRIBUTED_JAR_NAME=geomesa-hbase-distributed-runtime-hbase2_2.12-%%project.version%%.jar
```
Comment (elahrvivaz) [inline geomesa-hbase/geomesa-hbase-tools/bin/bootstrap-geomesa-hbase-aws.sh:62]:
```suggestion
DISTRIBUTED_JAR_NAME=geomesa-hbase-distributed-runtime-hbase2_%%scala.binary.version%%-%%project.version%%.jar
```

**Comment by elahrvivaz** [comment]
thanks for fixing this!

**Comment by ericsun95** [comment]
> thanks for fixing this!

You are welcome. Committed the suggested changes!

**Pull request merged**